tests/integration/test_aws/test_only_logs_after.py

```diff
@@ -285,21 +273,19 @@ def test_service_without_only_logs_after(
         callback=event_monitor.callback_detect_all_aws_err
     )
 
-    assert log_monitor.callback_result is None, ERROR_MESSAGES['error_found']
+    assert log_monitor.callback_result is None, ERROR_MESSAGE['error_found']
```

Comment:
`expected_results = metadata['expected_results']` is being declared inside this test but is never used, is that correct?

Reply:
No is not, it's unused coded and has been removed.